### DISCUSSION TITLE
Is Fortnite still a no-go?

### DISCUSSION THREAD

**Discussion by Shamanon** [lutris/lutris#5347] [Q&A] [answered]
I have seen some people say they got Fortnite to work with Lutris, but the Dev's say no... I am guessing the Dev's are correct unless something has changed... Don't want to download all 60 gigs of the game if it's not going to work.

**Comment by strycore** [comment, marked as answer]
Fortnite still has the same anticheat, so it's not going to work. Only way to play Fortnite without booting into Windows is to use a streaming service. I've tried GeForce Now and both Fortnite and Destiny 2 can be played. I haven't tried Xbox Cloud Gaming or Amazon Luna.

**Reply by strycore** [reply]
Xbox Cloud Gaming works well but it's controller only and the resolution is pretty low, probably because I'm using a free tier.

**Reply by Shamanon** [reply]
Luna works great if you already have a prime subscription and fast internet. I have prime, but my internet isn't fast enough once gameplay commences.
